screens/Scan/ScanScreen.tsx

```diff
@@ -151,13 +154,28 @@ export const ScanScreen: React.FC = () => {
   function displayStorageLimitReachedError(): React.ReactNode {
     return (
       !controller.isEmpty && (
-        <ErrorMessageOverlay
+        <MessageOverlay
           isVisible={
             controller.isMinimumStorageRequiredForAuditEntryLimitReached
           }
           translationPath={'ScanScreen'}
           error="errors.storageLimitReached"
-          onDismiss={() => navigation.navigate(BOTTOM_TAB_ROUTES.home)}
+          onBackdropPress={() => navigation.navigate(BOTTOM_TAB_ROUTES.home)}
+        />
+      )
+    );
+  }
+
+  function invalidQR(): React.ReactNode {
+    return (
+      !controller.isEmpty && (
+        <MessageOverlay
+          isVisible={controller.selectIsInvalid}
+          title={t('common.errors.genericError')}
+          // TODO: maybe add a message QR invalid
+          onButtonPress={() => navigation.navigate(BOTTOM_TAB_ROUTES.scan)}
+          buttonText={t('common:tryAgain')}
```

Comment:
I think literals are tailed with . not :
this **buttonText={t('common:tryAgain')}** should be like **buttonText={t('common.tryAgain')}**

Reply:
based on our conversations, this seems to be okay, let me know if further action is required on this